client/internal/httpsender.go

```diff
@@ -275,6 +275,13 @@ func (h *HTTPSender) receiveResponse(ctx context.Context, resp *http.Response) {
 	h.receiveProcessor.ProcessReceivedMessage(ctx, &response)
 }
 
+func (h *HTTPSender) SetHeartbeatInterval(duration time.Duration) {
+	// 0 is invalid for a http sender
```

Comment:
Can we return an `error` from this method to indicate invalid values? Negative values should also be invalid.